manifest.php

```diff
@@ -142,4 +147,10 @@
     'extra' => [
         'structures' => __DIR__ . DIRECTORY_SEPARATOR . 'actions' . DIRECTORY_SEPARATOR . 'structures.xml',
     ],
+    'containerServiceProviders' => [
+        CustomInteractionPostProcessingServiceProvider::class,
+        ItemsReferencesServiceProvider::class,
+        TestQtiServiceProvider::class,
+        TestSessionStateServiceProvider::class
```

Comment:
does target nccer extension support latest DI?

Reply:
no, this will not be used.

Reply:
may be it make sense to delete it, just do not confuse anyone in a future who will check compatibility of this extension and new DI. This declaration will allow make false conclusion about it